SpatialGDK/Source/SpatialGDKServices/Private/LocalDeploymentManager.cpp

```diff
@@ -266,9 +267,20 @@ void FLocalDeploymentManager::TryStartLocalDeployment(FString LaunchConfig, FStr
 	FSpatialGDKServicesModule& GDKServices = FModuleManager::GetModuleChecked<FSpatialGDKServicesModule>("SpatialGDKServices");
 	TWeakPtr<SSpatialOutputLog> SpatialOutputLog = GDKServices.GetSpatialOutputLog();
 
-	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](const FString& Output) {
+	RuntimeProcess->OnOutput().BindLambda([&, SpatialOutputLog](FString& Output) {
+		// Format and output the log to the editor window `SpatialOutputLog`
 		SpatialOutputLog.Pin()->FormatAndPrintRawLogLine(Output);
 
+		// Save the raw runtime output to disk.
+		if (RuntimeLogFileHandle)
+		{
+			// Always add a newline.
+			Output += LINE_TERMINATOR;
+
+			RuntimeLogFileHandle->Write((const uint8*)TCHAR_TO_ANSI(*Output), Output.Len());
+			RuntimeLogFileHandle->Flush();
```

Comment:
Do we need to be careful about file streaming performance issues here?

Reply:
You think flushing each line might be too much?
Can flush on a timer if you prefer